src/systems/filecoin_blockchain/storage_power_consensus/_index.md

```diff
@@ -55,38 +55,47 @@ Whenever comparing tickets is evoked in Filecoin, for instance when discussing s
+
 To sample a ticket for a given epoch n:
-- Set referenceTipsetOffset = 0
-- While true:
-    - Set referenceTipsetHeight = n - referenceTipsetOffset
-    - If blocks were mined at referenceTipsetHeight:
-        - ReferenceTipset = TipsetAtHeight(referenceTipsetHeight)
-        - Select the block in ReferenceTipset with the smallest final ticket, return its ticket (pastTicket).
-    - If no blocks were mined at referenceTipsetHeight:
-        - Increment referenceTipsetOffset
-        - (Repeat)
-- newRandomness = H(pastTicket || minerAddress || n)
+```text
+Set referenceTipsetOffset = 0
+While true:
+    Set referenceTipsetHeight = n - referenceTipsetOffset
+    If blocks were mined at referenceTipsetHeight:
+        ReferenceTipset = TipsetAtHeight(referenceTipsetHeight)
+        Select the block in ReferenceTipset with the smallest final ticket, return its value (pastTicket).
+    If no blocks were mined at referenceTipsetHeight:
+        Increment referenceTipsetOffset
+        (Repeat)
+newRandomness = H(pastTicket || n)
+```
 
 In english, this means two things:
+- Choose the smallest ticket in the Tipset if it contains multiple blocks.
 - When sampling a ticket from an epoch with no blocks, draw the min ticket from the prior epoch with blocks and concatenate it with
-    - the minerAddress of the miner using this input
     - the wanted epoch number
     - hash this concatenation for a usable ticket value
-- Choose the smallest ticket in the Tipset if it contains multiple blocks.
 
 See the `RandomnessAtEpoch` method below:
 {{< readfile file="block.go" code="true" lang="go" >}}
 
-The above means that ticket randomness is reseeded with every new block, but can indeed be derived by any miner for an arbitrary epoch number using a past epoch. However, this does not affect protocol security under Filecoin's clock synchrony assumption.
+The above means that ticket randomness is reseeded with every new block, but can indeed be derived by any miner for an arbitrary epoch number using a past epoch.
 
 {{<label ticket_generation>}}
-### Ticket generation
+### Randomness Ticket generation
 
 This section discusses how tickets are generated by EC for the `Ticket` field in every block header.
 
 At round `N`, a new ticket is generated using tickets drawn from the Tipset at round `N-1` (as shown below).
 
-The miner runs the prior ticket through a Verifiable Random Function (VRF) to get a new unique ticket which can later be derived for randomness (as shown above).
+The miner runs the prior ticket through a Verifiable Random Function (VRF) to get a new unique ticket which can later be derived for randomness (as shown above). The prior ticket is prepended with the ticket domain separation tag `1` and concatenated with the miner actor address (to ensure miners using the same worker keys get different randomness).
```

Comment:
Would prefer not to hard-code the DST value here.

Reply:
Different conception of Prose. I think the Prose should suffice when needed. Should be changed when necessary. It's more work but worth it to impl imo @anorth ?

Reply:
Agreed, but we shouldn't hard-code the number 1 in the prose since this might change -- should either pull it in from the code dynamically using hugo, or (much simpler) avoid mentioning the concrete value in the prose altogether, and just say it uses a DST (link to reference on DSTs).